### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone (for the main repo; adding outside repos should be fine) or `ldd`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/json-builder-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name. Tests -> testsuite.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include <stdint.h>
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by xansec** [comment]
Also, please do not upload such a large corpus.

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**